config/styles/blade-effects.md

```diff
@@ -26,6 +28,8 @@ Just read EFFECT as "do something during_XXXX"
 ## General effects
 ### EFFECT_NONE
 Never generated, used as defaults in some places to mean "no effect", but is otherwise not useful for anything.
+### CLASH_UPDATE
+*(OS 6.0+)* Internal functions that *shall not* be used on styles.
```

Comment:
it's not a function
Also *shall not* is a bit weird. I think I would describe it like this:

No effect in blade styles, but may be used in props and other places to detect when the strength of a clash increases after the EFFECT_CLASH has occurred.